tests/dataset_runner.py

```diff
@@ -1,8 +1,9 @@
 import os
+import time
```

Comment:
This is an unused import, but I only noticed while tests were running and didn't want to trigger an unnecessary build. I'll take care of it with any revisions.